Fix go vet errors

<!-- 🎉 Thanks for the bug report!! 🎉 -->

## Summary of Bug

When running go vet command it reports 3 not compliant statements
```bash
db/historydb/historydb_test.go:776:13: github.com/hermeznetwork/hermez-node/common.RollupVariables composite literal uses unkeyed fields
db/historydb/historydb_test.go:785:14: github.com/hermeznetwork/hermez-node/common.AuctionVariables composite literal uses unkeyed fields
db/historydb/historydb_test.go:802:15: github.com/hermeznetwork/hermez-node/common.WDelayerVariables composite literal uses unkeyed fields
``` 

### Expected Behavior

No error messages

### Steps to Reproduce

```bash
make test
```

### System information

N/A

### Additional Information:

N/A

--- a/Makefile
+++ b/Makefile
@@ -45,12 +45,12 @@ test: govet gocilint test-unit
 ## test-unit: Run all unit tests.
 test-unit:
 	@echo "  >  Running unit tests"
-	$(GOENVVARS) go test -race -p 1 -failfast -timeout 300s -v ./...
+	$(GOENVVARS) POSTGRES_PASS=$(POSTGRES_PASS) go test -race -p 1 -failfast -timeout 300s -v ./...
 
 ## test-api-server: Run the API server using the Go tests.
 test-api-server:
 	@echo "  >  Running unit tests"
-	$(GOENVVARS) FAKE_SERVER=yes go test -race -timeout 0 ./api -p 1 -count 1 -v
+	$(GOENVVARS) POSTGRES_PASS=$(POSTGRES_PASS) FAKE_SERVER=yes go test -race -timeout 0 ./api -p 1 -count 1 -v
 
 ## gofmt: Run `go fmt` for all go files.
 gofmt:

--- a/db/historydb/historydb_test.go
+++ b/db/historydb/historydb_test.go
@@ -772,40 +772,37 @@ func TestGetUnforgedL1UserTxs(t *testing.T) {
 }
 
 func exampleInitSCVars() (*common.RollupVariables, *common.AuctionVariables, *common.WDelayerVariables) {
-	//nolint:govet
 	rollup := &common.RollupVariables{
-		0,
-		big.NewInt(10),
-		12,
-		13,
-		[]common.BucketParams{},
-		false,
+		EthBlockNum:           0,
+		FeeAddToken:           big.NewInt(10),
+		ForgeL1L2BatchTimeout: 12,
+		WithdrawalDelay:       13,
+		Buckets:               []common.BucketParams{},
+		SafeMode:              false,
 	}
-	//nolint:govet
 	auction := &common.AuctionVariables{
-		0,
-		ethCommon.BigToAddress(big.NewInt(2)),
-		ethCommon.BigToAddress(big.NewInt(3)),
-		"https://boot.coord.com",
-		[6]*big.Int{
+		EthBlockNum:        0,
+		DonationAddress:    ethCommon.BigToAddress(big.NewInt(2)),
+		BootCoordinator:    ethCommon.BigToAddress(big.NewInt(3)),
+		BootCoordinatorURL: "https://boot.coord.com",
+		DefaultSlotSetBid: [6]*big.Int{
 			big.NewInt(1), big.NewInt(2), big.NewInt(3),
 			big.NewInt(4), big.NewInt(5), big.NewInt(6),
 		},
-		0,
-		2,
-		4320,
-		[3]uint16{10, 11, 12},
-		1000,
-		20,
+		DefaultSlotSetBidSlotNum: 0,
+		ClosedAuctionSlots:       2,
+		OpenAuctionSlots:         4320,
+		AllocationRatio:          [3]uint16{10, 11, 12},
+		Outbidding:               1000,
+		SlotDeadline:             20,
 	}
-	//nolint:govet
 	wDelayer := &common.WDelayerVariables{
-		0,
-		ethCommon.BigToAddress(big.NewInt(2)),
-		ethCommon.BigToAddress(big.NewInt(3)),
-		13,
-		14,
-		false,
+		EthBlockNum:                0,
+		HermezGovernanceAddress:    ethCommon.BigToAddress(big.NewInt(2)),
+		EmergencyCouncilAddress:    ethCommon.BigToAddress(big.NewInt(3)),
+		WithdrawalDelay:            13,
+		EmergencyModeStartingBlock: 14,
+		EmergencyMode:              false,
 	}
 	return rollup, auction, wDelayer
 }

--- a/eth/client.go
+++ b/eth/client.go
@@ -13,7 +13,7 @@ import (
 var errTODO = fmt.Errorf("TODO: Not implemented yet")
 
 const (
-	blocksPerDay = (3600 * 24) / 15
+	blocksPerDay = (3600 * 24) / 15 //nolint:gomnd
 )
 
 func max(x, y int64) int64 {

